ggml-aarch64.c

```diff
@@ -0,0 +1,2187 @@
+// SPDX-FileCopyrightText: Copyright 2024 Arm Ltd.
+#define GGML_COMMON_IMPL_C
+#include "ggml-common.h"
+
+#include "ggml-quants.h"
+#include "ggml-impl.h"
+
+#define GGML_COMMON_IMPL_C
+#include "ggml-common.h"
```

Comment:
```suggestion
```

Reply:
We removed this in the latest commit as suggested.